embedded-core/pom.xml

```diff
@@ -0,0 +1,120 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+      <groupId>org.sonarsource.sonarlint.core</groupId>
+      <artifactId>sonarlint-core-parent</artifactId>
+      <version>9.4-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>sonarlint-embedded-core</artifactId>
+
+    <properties>
+        <maven.compiler.source>17</maven.compiler.source>
```

Comment:
This is suspicious. We should rely on the settings defined in the parent, and not define them in each module.

Reply:
Agree. We don't seem to be setting these properties in other `pom.xml`s either so I removed them all.